src/tests/testUtils.ts

```diff
@@ -0,0 +1,12 @@
+import { screen, waitForElementToBeRemoved } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+async function loadingScreen() {
+  expect(screen.getByLabelText('loadingScreen')).toBeInTheDocument()
+  await waitForElementToBeRemoved(() =>
+    screen.queryByLabelText('loadingScreen'),
+  )
+}
+
+export * from '@testing-library/react'
```

Comment:
Why is this being exported?

Reply:
I believe it is so there is a single `import { xyz }  from '../path/to/testUtils'` instead of importing things from '@testing-library/react' itself.